api/src/main/java/com/velocitypowered/api/bossbar/BossBar.java

```diff
@@ -0,0 +1,135 @@
+package com.velocitypowered.api.bossbar;
+
+import com.velocitypowered.api.proxy.Player;
+import java.util.Collection;
+import net.kyori.text.Component;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * Represents a boss bar, which can be send to a (group of) player(s).
+ * <b>Boss bars only work on 1.9 and above.</b>
+ */
+public interface BossBar {
```

Comment:
Unless I'm mistaken, this class doesn't expose [boss bar flags](https://wiki.vg/Protocol#Boss_Bar)?

Reply:
I have not understand the magic of the flags in the packet, so i can't do that. But hey, 99% of the things are inside